### PR TITLE
Privacy page // Premium upsells

### PR DESCRIPTION
**Pull request by joe9663**
Creates Privacy page in dashboard and adds premium upsell notifications to All Forms, Tools, Entries and Settings pages.

### CODE REVIEW

**Comment by tonyr-wv** [comment]
@joe9663 @cssjoe  the privacy page shows a 30% discount it should be 10%

File: assets/css/admin.css [hunk 1]
```diff
@@ -1608,3 +1608,51 @@ ul.wpuf-form .wpuf-field-columns .wpuf-column-field-inner-columns .wpuf-column .
   justify-content: space-between;
   align-items: center;
 }
+
+/* Premium CTA */
+.wf-premium-cta {
```
Comment (cssjoe) [inline assets/css/admin.css:1613]: Class names should match the existing standard ".wpuf-".

File: assets/css/admin.css [hunk 2]
```diff
@@ -1608,3 +1608,51 @@ ul.wpuf-form .wpuf-field-columns .wpuf-column-field-inner-columns .wpuf-column .
...
+    }
+    .wf-premium-cta {
+        flex-direction: row;
+        align-items: center;
+        margin: 1em 0;
+    }
+}
+
+/* Privacy Page */
+.weforms-privacy-page {
```
Comment (cssjoe) [inline assets/css/admin.css:1651]: Class names should match the existing standard ".wpuf-".

File: assets/spa/components/weforms-page-privacy/template.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<p>If you own a web design, web development or digital marketing company, check out <a href="https://termageddon.com/agency-partners/" target="_blank">Termageddon's agency partner program</a>, where you can offer Termageddon licenses to your clients, helping them stay compliant with privacy laws.</p>
+
+<h2>How to add Privacy Policy consent to your forms</h2>
+
+<p>Adding a Privacy Policy consent checkbox is your forms is simple. All you will need is the WeForms plugin and an existing /privacy-policy page.</p>
+
+<ol>
+    <li>"Edit" the respective form that you'd like to add a consent checkbox to.</li>
+    <li>Click 'checkbox' so that it is added to the bottom of your form</li>
+    <li>"Edit" the consent checkbox, remove the 'Field Label', and replace 'Option' with 'INSERT YOUR "I AGREE" LANGUAGE HERE AND PROVIDE A LINK TO YOUR PRIVACY PPOLICY'.</li>
```
[the remark this answers is not in the document]
Comment (harryjackson1221) [reply assets/spa/components/weforms-page-privacy/template.php:24]: "What's the difference in the URLs for get-premium, upgrade-now, upgrade, and premium? They all appear to redirect to the pricing page. Is it better and more maintainable to use the same URL and have different UTM codes for tracking?"

The URL's are all different slugs that are redirecting to Pricing, and we are using 4 different UTM codes with different sources and mediums. We are future proofing it so we can create separate landing pages as needed.

**Pull request merged**